TestAssets/TestProjects/TestAppWithRuntimeOptions/project.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Did you check that we didn't have something like that already? Maybe not named like you did, but some test project that had runtimeoptions set in it?